README.md

```diff
@@ -3332,6 +3332,185 @@ These fields are optionally available:
 
 
 
+### wp signup
+
+Manages signups on a multisite installation.
+
+~~~
+wp signup
+~~~
+
+**EXAMPLES**
+
+    # List signups.
+    $ wp signup list
+    +-----------+------------+---------------------+---------------------+--------+------------------+
+    | signup_id | user_login | user_email          | registered          | active | activation_key   |
+    +-----------+------------+---------------------+---------------------+--------+------------------+
+    | 1         | bobuser    | bobuser@example.com | 2024-03-13 05:46:53 | 1      | 7320b2f009266618 |
+    | 2         | johndoe    | johndoe@example.com | 2024-03-13 06:24:44 | 0      | 9068d859186cd0b5 |
+    +-----------+------------+---------------------+---------------------+--------+------------------+
+
+    # Activate signup.
+    $ wp signup activate 2
+    Success: Signup activated. Password: bZFSGsfzb9xs
+
+    # Delete signup.
+    $ wp signup delete 3
+    Success: Signup deleted.
+
+
+
+### wp signup activate
+
+Activates a signup.
+
+~~~
+wp signup activate <signup>
+~~~
+
+**OPTIONS**
+
+	<signup>
+		Signup ID, user login, user email or activation key.
+
+**EXAMPLES**
+
+    # Activate signup.
+    $ wp signup activate 2
+    Success: Signup activated. Password: bZFSGsfzb9xs
+
+
+
+### wp signup delete
+
+Deletes a signup.
+
+~~~
+wp signup delete <signup>
+~~~
+
+**OPTIONS**
+
+	<signup>
+		Signup ID, user login, user email or activation key.
+
+**EXAMPLES**
+
+    # Delete signup.
+    $ wp signup delete 3
+    Success: Signup deleted.
+
+
+
+### wp signup get
+
+Gets details about the signup.
```

Comment:
Other commands use "a" instead of "the" ([example](https://github.com/wp-cli/entity-command/blob/22d726dd3ce09a7a70ca1ac2c935e2f0450894a4/README.md?plain=1#L2411)).
```suggestion
Gets details about a signup.
```